flow/project.py

```diff
@@ -1870,50 +2076,68 @@ def _print_progress(x):
                                 raise error
                             else:
                                 try:
-                                    results = self._fetch_status_in_parallel(
-                                        pool, cloudpickle, jobs, ignore_errors, cached_status)
+                                    results = self._fetch_labels_in_parallel(
+                                        pool, cloudpickle, fetched_jobs, ignore_errors,
+                                        cached_status)
                                 except self._PickleError as error:
                                     raise RuntimeError(
                                         "Unable to parallelize execution due to a pickling "
                                         "error: {}.".format(error))
-                        return list(tqdm(
-                            iterable=results,
-                            desc="Collecting job status info", total=len(jobs), file=err))
+                        label_results = _generate_results_with_tqdm(
+                            results, desc="Collecting job label info", len_itr=len(fetched_jobs))
+                        # TODO: We need to check in #366 whether ``group.aggregate``` method
+                        # can be pickled or not. It currently fails if we try to pickle.
+                        # The process is under development.
+                        op_results = _generate_results_with_tqdm('groups')
```

Comment:
This should be solved with static aggregation correct?

Reply:
Yes, this will be solved.
I will parallelize it.